include/Scene.h

```diff
@@ -19,6 +19,7 @@ class Scene {
     int GetGeometriesSize();
     Geometry* GetGeometry( int i );
     void ToggleSelectedGeometry( int i );
+    bool AnyGeometrySelected();
```

Comment:
Might be better to name this `HasAnyGeometrySelected()` to make it more obvious that it will return a bool but it's fine as is.